build-packages.sh

```diff
@@ -3,16 +3,16 @@
 usage()
 {
     echo "Builds the NuGet packages from the binaries that were built in the Build product binaries step."
-    echo "Usage: build-packages -BuildArch -BuildType [-portable]"
+    echo "Usage: build-packages -BuildArch -BuildType [-nonportable]"
     echo "BuildArch can be x64, x86, arm, arm64 (default is x64)"
     echo "BuildType can be release, checked, debug (default is debug)"
-    echo "-portable - build for Portable Distribution"
+    echo "-nonportable - build for Distro-specific RID"
```

Comment:
I agree with consistency but we could make them consistent by changing them all to nonportable as well. I'm OK with either option.